chainlink-node/main.star

```diff
@@ -97,7 +97,7 @@ def init_chain_connection(plan, args):
         return False, chain_name, chain_id, args["wss_url"], args["http_url"], None
     
     plan.print("Spinning up a local ETH chain and connecting to it")
-    eth_network_participants = eth_network_module.run(plan, args)
+    eth_network_participants, cl_genesis_timestamp = eth_network_module.run(plan, args)
```

Comment:
who is using this? i don't see the variable referenced elsewhere

Reply:
No one actually, it is just returned in the eth_network_module. should I just put a `_` instead?